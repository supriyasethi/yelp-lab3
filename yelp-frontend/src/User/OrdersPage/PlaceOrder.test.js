import React from 'react';
import Enzyme, { shallow, configure } from 'enzyme';
import PlaceOrder from './PlaceOrder.jsx'
import Adapter from 'enzyme-adapter-react-16';
Enzyme.configure({ adapter: new Adapter() });

describe('PlaceOrder', () => {
    test('it renders the two buttons', () => {
      const wrapper = shallow(<PlaceOrder />);
      expect(wrapper.find('.variant')).toMatchSnapshot();
    });
  });
