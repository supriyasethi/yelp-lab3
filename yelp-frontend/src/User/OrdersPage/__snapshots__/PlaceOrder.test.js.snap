// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PlaceOrder it renders the two buttons 1`] = `ShallowWrapper {}`;
